specifyweb/frontend/js_src/lib/components/QueryBuilder/helpers.ts

```diff
@@ -349,5 +345,31 @@ export function isSpecial(
       terminatingField.model.name === 'SpecifyUser';
     return endsWithSpecifyUser && includesUserValue;
   });
-  return containsOr || containsRelativeDate || containsSpecifyUserName;
+
+const containsRelativeDate = (
+  fieldSpecMapped: RA<readonly [QueryField, QueryFieldSpec]>
+) =>
+  fieldSpecMapped.some(
+    ([field, fieldSpec]) =>
+      field.filters.some(({ startValue }) => startValue.includes('today')) &&
+      fieldSpec.datePart === 'fullDate'
+  );
+
+// If contains modern fields/functionality set isFavourite to false, to not appear directly in 6
+export function isModern(query: SpecifyResource<SpQuery>): boolean {
```

Comment:
Since this function is globally exported, rename it to `isQueryModern` or `isModernQuery` or something like to make it less ambiguous

Reply:
Also, for comments that describe a function or a file, use the tsdoc syntax `/** your comment */` - see https://tsdoc.org/ (you can search for examples in our codebase)

PyCharm is able to recognize it and can render it beautifully. Other tools can also interpret it as it's machine readable